cfme/cloud/instance/__init__.py

```diff
@@ -124,11 +95,16 @@ def is_displayed(self):
                                .format(self.context['object'].provider.name) and
             self.sidebar.instances_by_provider.is_opened)
 
-    toolbar = View.nested(InstanceToolbar)
+    toolbar = View.nested(VMToolbar)
     sidebar = View.nested(InstanceAccordion)
-    entities = View.nested(InstanceEntities)
+    entities = View.nested(VMEntities)
     paginator = View.nested(PaginationPane)
 
+    @View.nested
```

Comment:
hm, I think this is a merge artifact, as this is in InstanceAccordion view in the sidebar above (L128/L99)  Will remove.